mcs/class/corlib/Test/System/TimeZoneInfoTest.cs

```diff
@@ -54,11 +54,11 @@ public static string MapTimeZoneId (string id)
 					return "New Zealand Standard Time";
 				case "Europe/Athens":
 					return "GTB Standard Time";
-				case "US/Eastern":
+				case "America/New_York":
 					return "Eastern Standard Time";
-				case "US/Central":
+				case "America/Chicago":
```

Comment:
we need to keep the `"US/Central"` case here since it is still used by `Bug_9664 ()` now.